src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -530,4 +530,20 @@ export class BrowsingContextImpl {
       },
     };
   }
+
+  async addPreloadScript(
+    params: Script.AddPreloadScriptParameters
+  ): Promise<Script.AddPreloadScriptResult> {
+    const result = await this.#cdpTarget.cdpClient.sendCommand(
+      'Page.addScriptToEvaluateOnNewDocument',
+      {
+        source: params.expression,
```

Comment:
`params.expression` is a function which has to be called (with bindings as arguments, but it's for later), while `Page.addScriptToEvaluateOnNewDocument. source` is a string literal (callFunction vs evaluate).